lib/trace_events.js

```diff
@@ -40,6 +40,11 @@ class Tracing {
     this[kEnabled] = false;
   }
 
+  /**
+   * Enables this `Tracing` object for the set of categories
```

Comment:
I'm hesitant about duplicating public API documents with `doc/api/trace_events.md`. This would mean that if we are going to update the doc about the public API, we will need to update both places. That sounds burdensome and easy to lose track of (since there are no means to enforce that).

Reply:
So should we just change the wording of the descriptions?